src/agent/onefuzz-telemetry/src/lib.rs

```diff
@@ -261,10 +261,10 @@ mod global {
     }
 }
 
-pub fn set_appinsights_clients(ikey: Option<Uuid>, tkey: Option<Uuid>) {
-    let instance_client = ikey.map(|k| TelemetryClient::new(k.to_string()));
-    let shared_client = tkey.map(|k| TelemetryClient::new(k.to_string()));
-    global::set_clients(instance_client, shared_client);
+pub fn set_appinsights_clients(instance_key: Option<Uuid>, microsoft_key: Option<Uuid>) {
```

Comment:
While we're here, maybe we should add newtypes/wrapper types for `Uuid` named `MicrosoftTelemetryKey` and `InstanceTelemetryKey` so we can't ever accidentally swap keys at call sites.